src/styles/event-form.less

```diff
@@ -860,3 +860,13 @@ form.booking-period {
     font-size: small;
   }
 }
+
+.form-age-range-button {
+  margin-bottom: 10px;
+
+  span{
+    font-size: small;
+    color: grey;
+    font-weight: 300;
+  }
+}
```

Comment:
Nit: There should probably be a newline at the end of this file?

Reply:
@bertramakers added a new line at the end 👍